e2e/app/page/admin-workspace-page.ts

```diff
@@ -0,0 +1,134 @@
+  }
+
+  getLoadWorkspaceButton(): Button {
+    return Button.findByName(this.page, { name: LinkText.LoadWorkspace });
+  }
+
+  async clickLoadWorkspaceButton(): Promise<void> {
+    const button = this.getLoadWorkspaceButton();
+    const navPromise = this.page.waitForNavigation({ waitUntil: ['load', 'networkidle0'] });
+    await button.click();
+    await navPromise;
+    await waitWhileLoading(this.page);
+  }
+
+  getLockWorkspaceButton(): Button {
+    return Button.findByName(this.page, { normalizeSpace: LinkText.LockWorkspace });
+  }
+
+  //click "Lock Workspace" button
+  async clickLockWorkspaceButton(): Promise<LockWorkspaceModal> {
+    const button = this.getLockWorkspaceButton();
+    await button.click();
+    const modal = new LockWorkspaceModal(this.page);
+    await modal.waitForLoad();
+    return modal;
+  }
+
+  // extract only the Workspace Namespace text for verification
+  async getWorkspaceNamespaceText(): Promise<string> {
+    const xpath = '//label[contains(normalize-space(),"Workspace Namespace")]//following-sibling::div';
+    const element = BaseElement.asBaseElement(this.page, await this.page.waitForXPath(xpath, { visible: true }));
+    const textContent = element.getTextContent();
+    return textContent;
+  }
+
+  // get the page header (workspace)
+  async getWorkspaceHeader(): Promise<string> {
+    const pageHeader = await this.page.waitForXPath('//div[contains(text(), "")]', { visible: true });
+    return getPropValue<string>(pageHeader, 'textContent');
+  }
+
+  getCloudStorageTable(): Table {
+    const selector = '//table[@class="p-datatable-scrollable-header-table"]';
+    return new Table(this.page, selector);
+  }
+
+  // get the column names of the cloud storage table
+  getcloudStorageColNames(): Promise<string[]> {
+    const cloudStorageTable = this.getCloudStorageTable();
+    const cloudStorageColNames = cloudStorageTable.getColumnNames();
+    return cloudStorageColNames;
+  }
+
+  // get the Notebook preview button
+  getNotebookPreviewButton(): Button {
+    return Button.findByName(this.page, { name: LinkText.Preview });
+  }
+
+  // click the Notebook preview button when it is enabled
+  async clickNotebookPreviewButton(): Promise<void> {
+    const button = this.getNotebookPreviewButton();
+    const navPromise = this.page.waitForNavigation({ waitUntil: ['load', 'networkidle0'] });
+    await button.click();
+    await navPromise;
+    await waitWhileLoading(this.page);
+  }
+
+  // get the text area to input the reason to access
+  getAccessReasonTextArea(): Textarea {
+    const selector =
+      '//label[contains(text(), "To preview notebooks, enter Access Reason (for auditing purposes)")]' +
+      '/following-sibling::textarea';
+    return new Textarea(this.page, selector);
+  }
+
+  // get the No Active Runtime Text to verify that no runtime is active
+  async getNoActiveRuntimeText(): Promise<string> {
```

Comment:
what does this return when there are active runtimes?  Does it make sense to add a check in here for the `noActiveRuntimeText` and return a `Promise<boolean>` instead?

Reply:
makes sense. Let me try that!

Reply:
makes sense. Let me try that!

Reply:
@jmthibault79 this function is only checking for the "no active run time running" text before creating the runtime and when there are active runtime - getRuntimeStatus() checks the runtime status. 
Since for every test run we are creating a new workspace and there is no ACTIVE runtime, I was thinking we could remove this function and the verification STEP. Thoughts!

Reply:
that seems OK to me